precompiles/parachain-staking/src/lib.rs

```diff
@@ -14,21 +14,26 @@
 // You should have received a copy of the GNU General Public License
 // along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Precompile to call parachain-staking runtime methods via the EVM
+//! Precompile to access parachain-staking pallet via the EVM
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
+#[cfg(test)]
+mod mock;
+#[cfg(test)]
+mod tests;
```

Comment:
So you've moved the tests from the runtime level to here right? That's great.

Reply:
Kind of, I added a subset of the runtime tests as unit tests. I think the runtime tests are still nice to have because they check that the precompiles work with the actual staking precompile address and the configured runtime. The unit tests are nice for checking basic functionality with the mock runtime (and a mock precompile address).